lib/events-reader.js

```diff
@@ -74,6 +80,22 @@ module.exports = function(harvesterApp) {
       var doc;
       doc = that.stream.read();
 
+      // If it is a doc add to kafka
+      if(doc !== null && (!doc.ns.includes('checkpoint'))) {
+        producer.init().then(function(){
+          return producer.send({
+            topic: 'darksouls',
+            partition: 0,
+            message: {
+              value: JSON.stringify(doc)
+            }
+          });
+        })
+        .then(function (result) {
+          console.log('>>> producer sent message');
+        });
+      }
+
       var promises = that.processDocHandlers(doc);
```

Comment:
You should call that fn here, in this promise chain. You probably want to replace line 99 with that call, OR you can add it to the chain of activity started by that "process doc handlers" chain.